### DISCUSSION TITLE
Users management

### DISCUSSION THREAD

**Discussion by metc** [komminarlabs/pulumi-influxdb#37] [Q&A] [answered]
Hello,

I am interested in [creating and managing users](https://docs.influxdata.com/influxdb/v2/admin/users/) in InfluxDB. How can I achieve this using Pulumi and Infrastructure as a code ?

Thanks you.
Christopher

**Comment by thulasirajkomminar** [comment]
Hey @metc 

I do not think the API supports creating users so you try with the authorisation tokens.

If the API does not support then I cannot implement it in the provider.

**Comment by thulasirajkomminar** [comment, marked as answer]
This is available now https://www.pulumi.com/registry/packages/influxdb/api-docs/user/
